.github/workflows/publish.yml

```
@@ -0,0 +1,33 @@
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - v[0-9]+.*
+      - test-release.*
+
+jobs:
+  publish-crates:
+    runs-on: ${{ matrix.os }}
```

actionlint (1.7.10)

property "os" is not defined in object type {}

(expression)